q2_taxa/assets/barplot/src/toolbar.js

```diff
@@ -153,22 +159,25 @@ export function addColorPicker(row, svg, data, dataMeta) {
   updateOpt.merge(enterOpt)
     .attr('value', d => d.name)
     .text(d => d.name);
+
   return grp;
 }
 
 export function addSortByPicker(row, svg, data, dataMeta) {
   const { metaData, sortedKeysReverse } = dataMeta;
   const grp = row.append('div').attr('class', 'col-lg-6 form-group sortByPicker');
   grp.append('label').text('Sort Samples By');
-  grp.append('button').html('<span class="glyphicon glyphicon-plus" aria-hidden="true"></span>')
-    .attr('class', 'btn btn-primary btn-xs')
-    .style('margin-left', '10px')
-    .on('click', () => {
-      const selects = grp.selectAll('.xCtrl');
-      if (selects.size() === metaData.length + sortedKeysReverse.length + 1) { return; }
-      const sel = _appendSortByPicker(grp, svg, data, dataMeta);
-      _sortBySelectOptions(sel, metaData, sortedKeysReverse, sortedKeysReverse[0]);
-    });
+  grp.append('a')
+      .on('click', () => {
+        const selects = grp.selectAll('.xCtrl');
+        if (selects.size() === metaData.length + sortedKeysReverse.length + 1) { return; }
+        const sel = _appendSortByPicker(grp, svg, data, dataMeta);
+        _sortBySelectOptions(sel, metaData, sortedKeysReverse, sortedKeysReverse[0]);
+      })
+    .append('span')
+      .attr('class', 'glyphicon glyphicon-plus-sign')
+      .attr('style', 'padding-left: 5px; cursor: pointer; cursor: hand;');
```

Comment:
This looks really good! Can you update the minus-sign button (for removing a sort) to match this style, too?